src/platformClient.js

```diff
@@ -29,14 +29,19 @@ class PlatformClient {
       }
       return config;
     }, error => {
+      let newError = error && error.response && {
```

Comment:
Should we include more of the request? I'm at least thinking of URL, method, request body and request I'd. Or did I miss that it's covered indirectly? Or should we just indirect the previous lot message that contains those details?

Just putting out thoughts that make certain error logs easier to consume.

Reply:
Everything is already being logged (full request and full response), right here we are only talking about the data that will literally be passed back to the caller of the `platformClient`.  Since they already know all their data, I can't think of a good reason to return duplicate information